README.md

```diff
@@ -1,8 +1,97 @@
-# hackshare
+# HackShare
 
-### How to Use
+A web app where fellows can teach other fellows the skills they believe they are good at.
 
-- Download the Repo to your local drive
-- Open `client` in terminal and type `npm start`
-- Open `server` in terminal and type `npm start`
-- Visit `localhost:3000`
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+What things you need to install the software and how to install them
+
+- [NodeJs](https://nodejs.org/)
+- [MongoDB](https://docs.mongodb.com/manual/administration/install-community/)
+
+### Installing
+
+A step by step series of examples that tell you how to get a development env running
+
+- Open Terminal and run
+
+```
+git clone https://github.com/eKhattak/hackshare.git
+```
+
+This will grab a local copy of repo
+
+- Open project directory in terminal
+
+```
+cd client
+npm install
+npm start
+```
+
+This will start React Application
+
+```
+Starting the development server...
+Compiled successfully!
+
+You can now view client in the browser.
+
+  Local:            http://localhost:3000
+  On Your Network:  http://192.168.10.6:3000
+
+Note that the development build is not optimized.
+To create a production build, use yarn build.
+```
+
+- Open project again in another terminal and run
+
+```
+cd server
+npm install
+npm start
+```
+
+This will start NodeJs Application
+
+```
+[nodemon] 2.0.4
+[nodemon] to restart at any time, enter `rs`
+[nodemon] watching path(s): *.*
+[nodemon] watching extensions: js,mjs,json
+[nodemon] starting `node app.js`
+```
+
+End with an example of getting some data out of the system or using it for a little demo
+
+## Deployment
+
+To be Added
+
+## Built With
+
+- [ReactJs](https://reactjs.org/) - An open-source JavaScript library for building user interfaces
+- [Nodejs](https://nodejs.org/) - An open-source, cross-platform, JavaScript runtime environment
+- [MongoDB](http://mongodb.com/) - A cross-platform document-oriented database program
+
+## Contributing
```

Comment:
Was this supposed to be Contribution Guidelines?

Reply:
Yes, we will have another `contribution.md` file which includes contribution guidelines.